src/components/contactBottomHeaderArea.scss

```diff
@@ -0,0 +1,173 @@
+.contact .header-area .top {
+    margin: var(--spacing-s) 0;
+    border-bottom: none;
+}
+
+.contact .header-area .bottom {
+    margin-top: 48px;
+    display: flex;
+    justify-content: space-between;
+}
+
+.contact .header-area .bottom .bottom__left {
+    flex: 0 1 60%;
+}
+
+.contact .header-area .bottom .bottom__left .form-row {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 52px;
+}
+
+.contact .header-area .bottom .bottom__left .form-row:last-of-type {
+    margin-bottom: 18px;
+}
+
+.contact .header-area .bottom .bottom__left .form-row>* {
+    width: 48%;
+}
+
+.contact .header-area .bottom .bottom__left .form-row input {
+    background: transparent;
+    border: none;
+    border-bottom: var(--border-thin-black);
+    font-family: var(--visueltRegular);
+    font-size: var(--headerFontSizeTertiary);
+    line-height: var(--paragraphLineHeight);
+    height: 48px;
+}
+
+.contact .header-area .bottom .bottom__left select {
+    appearance: none;
+    box-shadow: none;
+    outline-width: 0;
+    padding: 0 20px;
+    border-radius: 0px;
+    background: transparent;
+    background-image: url("data:image/svg+xml,<svg version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' width='24' height='24' viewBox='0 0 24 24'><path fill='%23444' d='M7.406 7.828l4.594 4.594 4.594-4.594 1.406 1.406-6 6-6-6z'></path></svg>");
+    background-position: 98% 50%;
+    background-repeat: no-repeat;
+    border: var(--border-thin-black);
+    font-family: var(--visueltRegular);
+    font-size: var(--headerFontSizeTertiary);
+    height: 65px;
+}
+
+.contact .header-area .bottom .bottom__left .form-row>.textarea-wrapper {
+    width: 100%;
+}
+
+.contact .header-area .bottom .bottom__left label {
+    display: block;
+    font-family: var(--visueltRegular);
+    font-size: var(--headerFontSizeTertiary);
+}
+
+.contact .header-area .bottom .bottom__left textarea {
```

Comment:
Can you add `font-family: var(--visueltRegular)` here as well so the font matches the other form elements?